app/services/feeds/check_item_medium_reply.rb

```diff
@@ -0,0 +1,37 @@
+# Checks if a Feedjira item represents a Medium reply or not
+module Feeds
+  class CheckItemMediumReply
+    MEDIUM_DOMAIN = "medium.com".freeze
```

Comment:
My only thought here would be to remove the specific domain from the class name and make the class accept the domain as an argument. That way if we had to do this for another domain we could use the same class. Not a blocker but a thought I had to make this class more vendor agnostic.

Reply:
Good idea! I initially wondered that there might have been a reason why the method was called `medium_reply?` and not just "reply?("medium.com")". I actually wanted to ping @maestromac about this but then I forgot. It seems due to some inconstencies in how Medium formats RSS feeds. Do you know?

Reply:
I do not know, @maestromac would be the one to ask or maybe @Zhao-Andy would know. I'm not very familiar with the RSS import stuff.

Reply:
Medium seems to be the only one that includes the user's comment in their RSS feed, for some reason.